[W13-3] Infinity Machine

fterh opened the pull request:
@case141
@DarrenDragonLee
@DoItTomorrow
@suriruhani

ewaldhew commented on README.adoc:
```diff
@@ -1,7 +1,7 @@
 
-https://travis-ci.org/se-edu/addressbook-level4[image:https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master[Build Status]]
+https://travis-ci.com/CS2103-AY1819S2-W13-3/main[image:https://travis-ci.com/CS2103-AY1819S2-W13-3/main.svg?branch=master[Build Status]]
 https://ci.appveyor.com/project/damithc/addressbook-level4[image:https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true[Build status]]
```
Update all the badges, not just the travis one.

ewaldhew replied:
Update all the badges, not just the travis one.

ewaldhew commented on docs/AboutUs.adoc:
```diff
@@ -4,53 +4,88 @@
-{empty}[http://github.com/yijinl[github]] [<<johndoe#, portfolio>>]
+=== Fabian Terh
+
+image::fterh.png[width="150", align="left"]
```
This photo is a bit too zoomed out, maybe crop it a bit to be more identifiable

ewaldhew commented on docs/UserGuide.adoc:
```diff
@@ -18,17 +18,14 @@ By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
 == Introduction
 
-AddressBook Level 4 (AB4) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB4 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB4 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Qui ...
+An application for efficient storing and retrieving of textual information on research material by allowing users to apply to it custom made tags and type for easy search and retrieval. Intended for tech-savvy university students who are able to use CLI and wish to organize and tag research information or any other data that they may wish to store.
```
Good

ewaldhew commented on docs/AboutUs.adoc:
```diff
@@ -4,53 +4,88 @@
+We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore]. +
+Tutorial W-13 Group 3: The Infinite Machine
 
 == Project Team
```
Roles: Who is in charge of which component?

ewaldhew commented on README.adoc:
```diff
@@ -14,19 +14,16 @@ ifndef::env-github[]
-* <<LearningOutcomes#, Learning Outcomes>>
 * <<AboutUs#, About Us>>
 * <<ContactUs#, Contact Us>>
 
```
Add acknowledgement to AB4

ewaldhew reviewed (commented):
Remember to update the project description on Github! The line on top that says "Address Book sample application"

Update the dev guide with your user stories, use cases, etc!

Tag features not yet implemented as [Coming in v2.0] in the user guide.

Update the line on top that says By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`

fterh committed:
Fix incorrect adoc include tag

fterh committed:
Close adoc tags

ToonDragon committed:
Massive update to the developer guide to properly explain implementation of the pin and unpin function

Minor enhancements to the user guide and the custom order implementation

ToonDragon committed:
Merge branch 'master' into master

ToonDragon committed:
Minor edits for image paths

ToonDragon committed:
Merge branch 'master' of https://github.com/DarrenDragonLee/main

case141 committed:
add restore command tests and update delete command and delete command tests

DoItTomorrow committed:
Merge branch 'master' into master

case141 committed:
Merge remote-tracking branch 'refs/remotes/origin/master'

case141 committed:
add redo and undo for recycle bin mode

suriruhani committed:
Merge pull request #237 from DarrenDragonLee/master

Massive update to the developer guide to properly explain implementation of the pin and unpin function

suriruhani committed:
Merge branch 'master' into fterh-update-docs

Commit:
Updated SampleDataUtil to match test data.

Commit:
Checkstyle Edits

DoItTomorrow committed:
Merge branch 'master' into master

Commit:
Merge branch 'master' of https://github.com/DoItTomorrow/main

ToonDragon committed:
Added sequence diagrams for pin and unpin commands

ToonDragon committed:
Merge branch 'master' into master

case141 committed:
add tests for undo and redo in recyclebin mode

case141 committed:
add tests for undo and redo in recyclebin mode

suriruhani committed:
Merge pull request #238 from DarrenDragonLee/master

Added sequence diagrams for pin and unpin commands

suriruhani committed:
Merge branch 'master' into fterh-update-docs

case141 committed:
add tests for undo and redo in restore command

suriruhani committed:
Merge pull request #236 from fterh/fterh-update-docs

Update documentation

suriruhani committed:
Merge pull request #37 from CS2103-AY1819S2-W13-3/master

merge

suriruhani committed:
Merge remote-tracking branch 'origin/master'

case141 committed:
add JsonDeletedSourcesStorageTests

suriruhani committed:
Merge branch 'master' into master

suriruhani committed:
update DG and rename variables in ListCommand

suriruhani committed:
update PPP

case141 committed:
add code to generate sampleDeletedData.json file

suriruhani committed:
add ListCommandSystemTest

Commit:
Removed old constructors for Source.

DoItTomorrow committed:
Merge branch 'master' into master

Commit:
Ignored 2 tests that are failling when run on their own but not when runned with all other tests

Commit:
Revert "Ignored 2 tests that are failling when run on their own but not when runned with all other tests"

This reverts commit 486159fda70bc7c5f700612a977786840ee2bfe2.

Commit:
Ignored 2 tests
Tests are passing when run on their own but not when runned with all other tests

case141 committed:
fix travis checkstyle errors

case141 committed:
Merge remote-tracking branch 'refs/remotes/origin/master'

Commit:
Removed unnecessary toString in EditCommand

Commit:
Checkstyle Edit

suriruhani committed:
Merge pull request #234 from case141/master

Fixed undo redo bug and Completed Tests for Recycle Bin

DoItTomorrow committed:
Merge branch 'master' into master

ToonDragon committed:
Merge pull request #22 from CS2103-AY1819S2-W13-3/master

Merge from Master

Commit:
Update to BiblioEditCommand to not break other tests. One test is current ignored.

Commit:
Checkstyle Edits

Commit:
Checkstyle Edits

suriruhani committed:
Merge pull request #38 from CS2103-AY1819S2-W13-3/master

merge

suriruhani committed:
Merge pull request #239 from suriruhani/master

List and Search: Kinda everything

suriruhani committed:
Merge pull request #39 from CS2103-AY1819S2-W13-3/master

Merge pull request #239 from suriruhani/master

fterh committed:
Update LogicClassDiagram

fterh committed:
Resize AliasManagerClassDiagram

fterh committed:
Update PPP

Commit:
Added test for SampleDataUtil

DoItTomorrow committed:
Merge branch 'master' into master

Commit:
Removed unnecessary import in util/TypicalSources

Commit:
Merge branch 'master' of https://github.com/DoItTomorrow/main

Commit:
Checkstyle Edits

Commit:
Removed unnecessay import in util/TypicalSources.java

Commit:
Readded accidental removal of import

DoItTomorrow committed:
Merge pull request #227 from DoItTomorrow/master

Added test for BiblioEditCommand

fterh committed:
Remove projects section from PPP

fterh committed:
Update and resize AliasManagerSequencDiagram

fterh committed:
Resize AliasManagerMetaCommandSequenceDiagram.png

Commit:
Updated sample and test source's biblioFields
Changed from Placeholder * to Foo*
Serves to differentiate entries from placeholders used by BilbioCommand more.

suriruhani committed:
minor change to search logic and DG update

suriruhani committed:
Merge pull request #40 from CS2103-AY1819S2-W13-3/master

merge

suriruhani committed:
Merge remote-tracking branch 'origin/master'

suriruhani committed:
add MT to PPP

suriruhani committed:
update DG

ToonDragon committed:
Added UI component for the Pinned sources

Minor logic improvements to source, delete, pin, unpin and order to account for new modifications

ToonDragon committed:
Merge branch 'master' into master

ToonDragon committed:
Removed Whitespaces and minor typo edits

ToonDragon committed:
Updated developer docs for new UI component and added new tests for pinned sources

suriruhani committed:
update tests

case141 committed:
Merge pull request #38 from CS2103-AY1819S2-W13-3/master

merge

ToonDragon committed:
Added new tests and sligitly improved logic

case141 committed:
add userguide mode explanation

case141 committed:
Merge remote-tracking branch 'refs/remotes/origin/master'

suriruhani committed:
update Storage Component Class diagream

suriruhani committed:
Merge pull request #241 from DarrenDragonLee/master

Added UI component for the Pinned sources

case141 committed:
restructured userguide

suriruhani committed:
Merge branch 'master' into master

suriruhani committed:
update Storage Component Class diagram*

case141 committed:
change structure

suriruhani committed:
undo redo diagrams update

case141 committed:
update userguide.adoc

case141 committed:
update userguide.adoc

case141 committed:
update userguide.adoc

case141 committed:
update userguide.adoc

case141 committed:
update userguide.adoc

ToonDragon committed:
Minor enhancement to the pin command to limit users to 5 pinned sources only

Complete with necessary tests and user guide improvements

Commit:
Change biblioFields access in Source to private final

ToonDragon committed:
Merge branch 'master' into master

suriruhani committed:
undo redo diagrams update

suriruhani committed:
Merge remote-tracking branch 'origin/master'

case141 committed:
update userguide.adoc

Commit:
Removed unnecessary import

ToonDragon committed:
Removed ONE whitespace

case141 committed:
update userguide.adoc

case141 committed:
update userguide.adoc

case141 committed:
update userguide.adoc

case141 committed:
update userguide.adoc

case141 committed:
update userguide.adoc

DoItTomorrow committed:
Merge branch 'master' into master

DoItTomorrow committed:
Merge pull request #243 from DoItTomorrow/master

Changed BiblioFields access in Source to Private

Commit:
Update to Userguide entries for biblio and biblioEdit commands

Commit:
Merge branch 'master' of https://github.com/DoItTomorrow/main

case141 committed:
Merge pull request #39 from CS2103-AY1819S2-W13-3/master

merge

ToonDragon committed:
Merge branch 'master' into master

fterh committed:
Merge branch 'master' into fterh-update-docs

Commit:
Update to Userguide entries for Biblio and BiblioCommand

suriruhani committed:
add search command seq. diagram

suriruhani committed:
Merge pull request #41 from CS2103-AY1819S2-W13-3/master

merge

suriruhani committed:
Merge remote-tracking branch 'origin/master'

fterh committed:
Update SDforDeletePerson.png to SDforDeleteSource.png

fterh committed:
Merge remote-tracking branch 'origin/fterh-update-docs' into fterh-update-docs

case141 committed:
update userguide recyclebin section

case141 committed:
Merge remote-tracking branch 'refs/remotes/origin/master'

case141 committed:
update userguide recyclebin section

case141 committed:
update userguide

case141 committed:
update userguide

suriruhani committed:
add pic to PPP

case141 committed:
update userguide

ToonDragon committed:
Added new sources for the demo

suriruhani committed:
add ListCommand seq. diagram

ToonDragon committed:
Revert "Added new sources for the demo"

This reverts commit 0efe044b1924e8f15755e2d24e73aa407e8fce85.

suriruhani committed:
add ListCommand seq. diagram

ToonDragon committed:
Merge pull request #242 from DarrenDragonLee/master

Minor enhancement to the pin command to limit users to 5 pinned sources at one time

case141 committed:
update userguide

suriruhani committed:
Merge branch 'master' into master

case141 committed:
Merge branch 'master' into master

ToonDragon committed:
Added new sources for the demo

ToonDragon committed:
Merge branch 'master' into master

suriruhani committed:
Merge pull request #240 from suriruhani/master

Updates

Commit:
Update to User and Dev guide entries on Biblio.

case141 committed:
Merge branch 'master' into master

suriruhani committed:
Merge branch 'master' into master

Commit:
Formatting adjustements to Biblio and BiblioEdit entries in DG

suriruhani committed:
add diagrams

case141 committed:
Merge pull request #245 from case141/master

Add Recycle Bin Mode and Restructured User Guide

suriruhani committed:
Merge branch 'master' into master

case141 committed:
rename PPP file and updating aboutus.adoc

suriruhani committed:
Merge pull request #246 from DarrenDragonLee/master

Added new sources for the demo

case141 committed:
Merge remote-tracking branch 'refs/remotes/origin/master'

suriruhani committed:
Merge pull request #42 from CS2103-AY1819S2-W13-3/master

merge

suriruhani committed:
delete dummy command greet

suriruhani committed:
Merge remote-tracking branch 'origin/master'

case141 committed:
update aboutus.adoc

case141 committed:
Merge pull request #40 from CS2103-AY1819S2-W13-3/master

merge

case141 committed:
Merge branch 'master' of https://github.com/case141/main

fterh committed:
Merge branch 'master' into fterh-update-docs

fterh committed:
Merge pull request #244 from fterh/fterh-update-docs

Update docs

suriruhani committed:
add typical sources

case141 committed:
updated issamesource to detect author field

case141 committed:
Merge pull request #41 from CS2103-AY1819S2-W13-3/master

merge

case141 committed:
explain duplicate sources in user guide

case141 committed:
update userguide

case141 committed:
update userguide

case141 committed:
edit duplicate sources test

suriruhani committed:
fixed breaking tests

suriruhani committed:
fixed breaking tests

suriruhani committed:
ignore failing json serializable source manager test

suriruhani committed:
Merge pull request #43 from CS2103-AY1819S2-W13-3/master

merge

case141 committed:
add uiclassdiagram.png

case141 committed:
update userguide

suriruhani committed:
Merge pull request #248 from suriruhani/master

Fixed adding typical sources [with 1 ignored test]

Commit:
Updated DG entries on Biblio feature

suriruhani committed:
clean list command

suriruhani committed:
clean list command parser

suriruhani committed:
clean search command

suriruhani committed:
no author name msg

suriruhani committed:
clean SCKP

fterh committed:
Fix inaccuracy in UserGuide

suriruhani committed:
clean list command test

Commit:
Update to Biblio feature on Developer and User guide.

suriruhani committed:
clean list command parser

fterh committed:
Fix typos/bugs

case141 committed:
update PPP

suriruhani committed:
clean list command test

case141 committed:
update PPP

Commit:
Update to biblio relate features for userguide.

fterh committed:
Merge pull request #249 from fterh/fterh-patch

Patch code

case141 committed:
update ppp

suriruhani committed:
clean search command parser test

case141 committed:
update ppp

suriruhani committed:
clean aboutus

suriruhani committed:
clean contact us

case141 committed:
update dev guide undoredo

suriruhani committed:
clean README

case141 committed:
Merge branch 'master' into master

case141 committed:
Merge pull request #247 from case141/master

Update DeveloperGuide with RecycleBin and Miscellaneous changes

case141 committed:
update dev guide

case141 committed:
Merge remote-tracking branch 'refs/remotes/origin/master'

case141 committed:
Merge branch 'master' into master

Commit:
Updated PPP

suriruhani committed:
clean PPP

DoItTomorrow committed:
Merge branch 'master' into master

ToonDragon committed:
Added a few new figures and finalised the PPP

ToonDragon committed:
Merge branch 'master' of https://github.com/DarrenDragonLee/main

ToonDragon committed:
Merge branch 'master' into master

Commit:
Updated Code contributions in PPP

Commit:
Merge branch 'master' of https://github.com/DoItTomorrow/main

Commit:
Formatting changes to PPP

suriruhani committed:
clean UG

ToonDragon committed:
Added tip to tell users not to add or edit same source

ToonDragon committed:
Added picture to PPP

ToonDragon committed:
Merge pull request #252 from DarrenDragonLee/master

Added a few new figures and finalised the PPP

fterh committed:
Update fterh.png

fterh committed:
Update responsibilities

fterh committed:
Merge branch 'master' into patch-about

suriruhani committed:
Merge pull request #44 from CS2103-AY1819S2-W13-3/master

merge

suriruhani committed:
Merge remote-tracking branch 'origin/master'

suriruhani committed:
Merge branch 'master' of /Users/suriruhani/Desktop/CS2103/main with conflicts.

fterh committed:
Merge pull request #253 from fterh/patch-about

Patch about

suriruhani committed:
Merge branch 'master' into master

suriruhani committed:
fixes

suriruhani committed:
restored DG

suriruhani committed:
restored DG

Commit:
Added process diagram
Minor formatting edits

case141 committed:
update devguide

suriruhani committed:
Merge pull request #254 from suriruhani/master

Updates

case141 committed:
update dev guide and ppp

ToonDragon committed:
Minor edit

ToonDragon committed:
More edits

Commit:
Update to AboutUs

DoItTomorrow committed:
Merge branch 'master' into master

ToonDragon committed:
Edits

ToonDragon committed:
Edit again

DoItTomorrow committed:
Merge pull request #251 from DoItTomorrow/master

Updates to documentation

ToonDragon committed:
Merge branch 'master' into master

suriruhani committed:
update

suriruhani committed:
Merge pull request #45 from CS2103-AY1819S2-W13-3/master

merge

suriruhani committed:
Merge remote-tracking branch 'origin/master'

ToonDragon committed:
Added EOF

suriruhani committed:
Merge pull request #256 from suriruhani/master

DG update

ToonDragon committed:
Merge branch 'master' of https://github.com/DarrenDragonLee/main

suriruhani committed:
Merge branch 'master' into master

ToonDragon committed:
Merge branch 'master' into master

case141 committed:
add diagram

case141 committed:
Merge remote-tracking branch 'refs/remotes/origin/master'

ToonDragon committed:
Merge pull request #255 from DarrenDragonLee/master

Minor edits to fit 10 pages

case141 committed:
updates to dev guide

case141 committed:
Merge branch 'master' into master

case141 committed:
update

case141 committed:
Merge remote-tracking branch 'refs/remotes/origin/master'

case141 committed:
Merge pull request #250 from case141/master

Completed DeveloperGuide updates

case141 committed:
portfolio

case141 committed:
Merge branch 'master' into master

suriruhani committed:
Merge pull request #258 from case141/master

portfolio
